hw/dv/tools/xcelium/xcelium_shm.tcl

```diff
@@ -5,7 +5,7 @@
 if {[info exists ::env(WAVES)]} {
   if {$::env(WAVES) == 1} {
     database -open -default -shm $::env(DUMP_FILE)
-    probe -all -depth all -shm
+    probe tb -all -depth all -shm
```

Comment:
Please change this to `$::env(TB_TOP)` instead - it is passed as an env variable by `dvsim`.

Reply:
@sriyerg fixed. thank you for pointing this out.